dockerbuild/Dockerfile-backend

```diff
@@ -63,8 +63,33 @@ WORKDIR /modeldb/server
 RUN ../scripts/gen_thrift_file.sh java '../thrift/ModelDB.thrift' './src/main/thrift/' && \
     mvn clean compile -Dthrift_version=0.10.0
 
-# This would be better as an ENTRYPOINT with 'mongo' the CMD param, but
-# `exec mvn exec:java ...` leads to a classpath issue that isn't immediately
+RUN mvn package -Dthrift_version=${THRIFT_VERSION} -Dmaven.test.skip=true
+
+FROM openjdk:8
+
+WORKDIR /modedlb/server
+
+EXPOSE 6543
+
+
+COPY server/target/modeldb-*.jar /modeldb/modeldb.jar
```

Comment:
This line needs the `--from=builder` flag.

This exposes an error in the modeldb configuration: The `.dockerignore` file needs to have the maven build output directories added. (`server/target/` especially).

Reply:
done